tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/TracerSettingsSnapshotGenerator/TracerSettingsSnapshot.g.cs

```diff
@@ -15,6 +15,7 @@ internal TracerSettingsSnapshot(Datadog.Trace.Configuration.TracerSettings setti
         ServiceNameInternal = settings.ServiceNameInternal;
         ServiceVersionInternal = settings.ServiceVersionInternal;
         TraceEnabledInternal = settings.TraceEnabledInternal;
+        ExperimentalAppsecStandaloneEnabledInternal = settings.ExperimentalAppsecStandaloneEnabledInternal;
```

Comment:
Once you remove `[ConfigKey]` you can (and should) remove all the changes in this file too!